app/site/_data/filters.json

```diff
@@ -0,0 +1,29 @@
+{
+    "organizations": [
```

Comment:
These are added manually. In the future, we should pull these from another file, or an API ideally.